backend/models.py

```
@@ -1,5 +1,5 @@
 from pydantic import BaseModel, Field, computed_field
-from typing import List, Optional, Dict
+from typing import List, Optional, Dict, Any, Union
```

Flake8 (7.2.0)

[error] 'typing.Union' imported but unused

(F401)